Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dto/CreateQuestionDTO.java

```diff
@@ -1,25 +0,0 @@
-package de.hft_stuttgart.winf.proj2.sp.backend.dto;
-
```

Comment:
why was this class deleted? isnt it used by other parts of the application?

Reply:
Class has been renamed